LBH.AdultSocialCare.Api/V1/Domain/Payments/DraftPayRunCreationDomain.cs

```diff
@@ -1,18 +1,14 @@
-using System;
-using System.Collections.Generic;
-using System.Linq;
-using System.Threading.Tasks;
 using Common.AppConstants.Enums;
-using LBH.AdultSocialCare.Api.Attributes;
 using LBH.AdultSocialCare.Api.V1.AppConstants.Enums;
-using LBH.AdultSocialCare.Api.V1.Infrastructure.Entities.Payments;
+using System;
 
 namespace LBH.AdultSocialCare.Api.V1.Domain.Payments
 {
     public class DraftPayRunCreationDomain
     {
         public PayrunType Type { get; set; }
         public PayrunStatus Status { get; set; }
+        public DateTimeOffset? PaidFromDate { get; set; }
         public DateTimeOffset PaidUpToDate { get; set; }
         public DateTimeOffset StartDate { get; set; }
         public DateTimeOffset EndDate => PaidUpToDate;
```

Comment:
What will happen if the PayrunType is AdHoc and released? In that case, PaidUpToDate will be null. 

If the FE always use PaidUpToDate, it is ok but in that situation remove one of them to avoid confusion. Or keep both but don't assigned the value.

Reply:
![image](https://user-images.githubusercontent.com/81692554/141328373-04521ba5-c8c9-4f92-99f4-618728a595d9.png)